src/coreclr/vm/peimagelayout.cpp

```diff
@@ -94,15 +101,21 @@ PEImageLayout* PEImageLayout::Map(PEImage* pOwner)
     }
     CONTRACT_END;
 
-    PEImageLayoutHolder pAlloc(new MappedImageLayout(pOwner));
+    PEImageLayoutHolder pAlloc = pOwner->GetUncompressedSize() ?
```

Comment:
Similar as the other place - I would prefer `pOwner->GetUncompressedSize() != 0` - using numbers as booleans is confusing.